integration_tests/sorted1.py

```diff
@@ -0,0 +1,13 @@
+from ltypes import f64
+
+def f():
+    x: list[f64] = [35.9, 24.9, 23.1, 4223.5, 64.34, 3095.34]
+    x = sorted(x)
+    ans: list[f64] = [23.1, 24.9, 35.9, 64.34, 3095.34, 4223.5]
+    eps: f64 = 1e-12
+    e: f64
+    i: i32
+    for i in range(len(x)):
+        assert abs(x[i]-ans[i]) < eps
+
+f()
```

Comment:
Take a larger list of integers say 100-200, fill the list as `[n, n - 1, n - 2, ...., 0]` and then sort it. Then check for correctness. May be benchmark your algorithm separately for comparisons with C++ sorting algorithms. Let's see where we stand.

Reply:
I think that would be fine, as long as it runs quickly (=immediately).